### PR TITLE
Refactor Config Fixtures

### PR DESCRIPTION
**Pull request by BlakeASmith**
- Adds `ConfigFixtureGroup` class
    - `.add_options` method for adding click options
- Ensures config options are loaded for CLI

### CODE REVIEW

File: tests/virtool_workflow/analysis/test_skewer/test_trimming_feature.yml [hunk 1]
```diff
@@ -175,31 +175,31 @@ bases:
   - 38.0
 - - 37.0
   - 38.0
-  - 38.0
+  - 37.5
```
Comment (igboyes) [inline tests/virtool_workflow/analysis/test_skewer/test_trimming_feature.yml:178]: Want to double-check that changes to these numbers are intended.

File: virtool_workflow/config/loading.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from virtool_workflow.fixtures.scope import FixtureScope
+from virtool_workflow import hooks
+
+from .fixtures import options
+
+
+async def load_config(scope=None, hook=None, **kwargs):
```
Comment (igboyes) [inline virtool_workflow/config/loading.py:7]: No types here?

File: tests/virtool_workflow/analysis/test_skewer/test_trimming_feature.yml (hunk 1 above)
Comment (BlakeASmith) [reply tests/virtool_workflow/analysis/test_skewer/test_trimming_feature.yml:178]: The `test_trimming_feature.yml` file is produced by `pytest-regressions`. I'm not sure what caused the change in output. 

The purpose of the file is to ensure that the same `fastqc` output is produced. So as long as the test suite is passing it's probably fine? 

I can look into it further if you like.

File: virtool_workflow/config/loading.py (hunk 2 above)
Comment (BlakeASmith) [reply virtool_workflow/config/loading.py]: Types added!

File: tests/virtool_workflow/analysis/test_skewer/test_trimming_feature.yml (hunk 1 above)
Comment (BlakeASmith) [reply tests/virtool_workflow/analysis/test_skewer/test_trimming_feature.yml]: Not sure of the cause, but the issue seems to be resolved now. The changes are no longer showing in the `Files Changed` section.

Comment (igboyes) [reply tests/virtool_workflow/analysis/test_skewer/test_trimming_feature.yml]: Doesn't that mean the test will pass even if that file is changing all the time. The test is supposed to ensure the file stays the same. I will worry if it changes again in the next PR.

Comment (BlakeASmith) [reply tests/virtool_workflow/analysis/test_skewer/test_trimming_feature.yml]: > Doesn't that mean the test will pass even if that file is changing all the time. The test is supposed to ensure the file stays the same. I will worry if it changes again in the next PR.

No the file is only reproduced if it doesn't already exist, and in the case that that happens the test does fail. So the correct file can only change if it is explicitly modified and committed to the repo.

**Review by igboyes — APPROVED** [review]

**Pull request merged**